tools/rgrnastar/README.rst

```diff
@@ -3,7 +3,7 @@ System Requirements
 ===================
 
 -  **Memory**: To run efficiently, RNA-STAR requires enough free memory to
-   hold the SA-indexed reference genome in RAM. For Human Genome hg19 this is
-   index about 27GB and running RNA-STAR requires approximately ~30GB of RAM.
-   For custom genomes the rule of thub is to multiply the size of the 
-   reference FASTA file with 9 to get the estimated required amount of RAM.
+   hold the SA-indexed reference genome in RAM. For Human Genome hg19 this
+   index is about 27GB and running RNA-STAR requires approximately ~30GB of RAM.
```

Comment:
Hi @dpryan79 thanks for all this work!!  The readme is only shown during installation. Would it be desired to copy pasta this section into here: https://github.com/dpryan79/tools-iuc/blob/90e25c02f890a89d9abb386b92b15632457be563/data_managers/data_manager_star_index_builder/data_manager/rna_star_index_builder.xml#L87 ?